src/nyx/scan_fastloader_way.cpp

```diff
@@ -91,6 +91,23 @@ namespace Nyxus
 		}  
 	}
 
+	std::string group_thousands (unsigned int x)
```

Comment:
Seems like this is not used anywhere in the code.

Reply:
Removed from PR